File: Foundation/Scratch/Getting Started/c++/l002countDigit.cpp
#include<iostream>
using namespace std;
int countDigit(int n){
        int ctr = 0;
        while(n!=0){
            n /= 10;
            ctr++;
        }
        return ctr;
    }
int main(){
        int n; cin>>n; 
        cout<<("Hello")<<endl;
        cout<<countDigit(n)<<endl;
    }

File: Foundation/Scratch/Getting Started/java/l002countDigit.java
import java.util.Scanner;
public class l002countDigit{
    public static Scanner scn = new Scanner(System.in);
    public static int countDigit(int n){
        int ctr = 0;
        while(n!=0){
            n /= 10;
            ctr++;
        }
        return ctr;
    }
    public static void main(String[] args){
        int n = scn.nextInt();
        System.out.println(countDigit(n));
    }
}
